integration/create_test.go

```diff
@@ -26,6 +27,57 @@ import (
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
+func TestCreateStressPostgres(t *testing.T) {
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1206")
+	setup.SkipForTigrisWithReason(t, "Tigris needs a schema for collection creation")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	db := collection.Database()
+
+	collNum := 10
```

Comment:
```suggestion
	const collNum = runtime.GOMAXPROCS() * 10
```

Reply:
This is unclear to me, could you explain the idea behind it?